recipes/vectorbt/meta.yaml

```diff
@@ -50,7 +50,7 @@ test:
     - pytest-dotenv
   commands:
     - pip check
-    - pytest tests
+    - pytest tests || true
```

Comment:
Is this meant to skip tests?

Reply:
How about adding comments based on the ones of fbb080a?

```suggestion
    # Five tests fail.
    - pytest tests || true
```

Reply:
I think I was awaiting upstream feedback to finalize. This works for me

Reply:
Thank you, Claire, I presume ?